include/mme-app/mmeStates/detachWfPurgeResp.h

```diff
@@ -1,61 +1,61 @@
- /*
- * Copyright 2019-present, Infosys Limited.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
- 
- /******************************************************
- * detachWfPurgeResp.h
- * This is an auto generated file.
- * Please do not edit this file.
- * All edits to be made through template source file
- * <TOP-DIR/scripts/SMCodeGen/templates/stateMachineTmpls/state.h.tt>
- ******************************************************/
- 
-#ifndef __DetachWfPurgeResp__
-#define __DetachWfPurgeResp__
-
-#include "state.h"
-
-namespace mme {
-
-	class DetachWfPurgeResp : public SM::State
-	{
-		public:
-			/******************************************
-			* Instance 
-			*    Creates static instance for the state
-			*******************************************/
-			static DetachWfPurgeResp* Instance();
-
-			/****************************************
-			* DetachWfPurgeResp
-			*    Destructor
-			****************************************/
-			~DetachWfPurgeResp();			
-			
-			/******************************************
-			* initialize
-			*  Initializes action handlers for the state
-			* and next state
-			******************************************/
-			void initialize();
-	
-		private:
-			/****************************************
-			* DetachWfPurgeResp
-			*    Private constructor
-			****************************************/
-			DetachWfPurgeResp();  
-	};
-};
-#endif // __DetachWfPurgeResp__
+ /*
+ * Copyright 2019-present, Infosys Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+ /******************************************************
+ * detachWfPurgeResp.h
+ * This is an auto generated file.
+ * Please do not edit this file.
+ * All edits to be made through template source file
+ * <TOP-DIR/scripts/SMCodeGen/templates/stateMachineTmpls/state.h.tt>
```

Comment:
Looks to be autogenerated file. Please update short copyright license.

Reply:
Done.